examples/filesystem/main.c

```diff
@@ -26,6 +26,16 @@
 #include "shell.h"
 #include "board.h" /* MTD_0 is defined in board.h */
 
+#if  defined(MTD_0) && \
```

Comment:
`MODULE_MTD_SDCARD` can still define `MTD_0`, so we would need to move this lower.

Reply:
Moved it to below the filesystem/mtd handling